src/Native/build-native.sh

```diff
@@ -150,8 +150,8 @@ __UnprocessedBuildArgs=
 __CrossBuild=0
 __ServerGC=0
 __VerboseBuild=false
-__ClangMajorVersion=3
-__ClangMinorVersion=5
+__ClangMajorVersion=0
```

Comment:
Interesting. I was convinced it is the other way in the build.sh in coreclr. Obviously I was wrong.